DataFormats/CSCDigi/interface/CSCALCTDigi.h

```diff
@@ -35,50 +39,56 @@ class CSCALCTDigi {
   void setValid(const int valid) { valid_ = valid; }
 
   /// return quality of a pattern
-  int getQuality() const { return quality_; }
+  uint16_t getQuality() const { return quality_; }
 
   /// set quality
   void setQuality(const int quality) { quality_ = quality; }
 
   /// return Accelerator bit
   /// 1-Accelerator pattern, 0-CollisionA or CollisionB pattern
-  int getAccelerator() const { return accel_; }
+  uint16_t getAccelerator() const { return accel_; }
 
   /// set accelerator bit
   void setAccelerator(const int accelerator) { accel_ = accelerator; }
 
   /// return Collision Pattern B bit
   /// 1-CollisionB pattern (accel_ = 0),
   /// 0-CollisionA pattern (accel_ = 0)
-  int getCollisionB() const { return patternb_; }
+  uint16_t getCollisionB() const { return patternb_; }
 
   /// set Collision Pattern B bit
   void setCollisionB(const int collision) { patternb_ = collision; }
 
   /// return key wire group
-  int getKeyWG() const { return keywire_; }
+  uint16_t getKeyWG() const { return keywire_; }
 
   /// set key wire group
   void setKeyWG(const int keyWG) { keywire_ = keyWG; }
 
   /// return BX - five low bits of BXN counter tagged by the ALCT
-  int getBX() const { return bx_; }
+  uint16_t getBX() const { return bx_; }
 
   /// set BX
   void setBX(const int BX) { bx_ = BX; }
 
   /// return track number (1,2)
-  int getTrknmb() const { return trknmb_; }
+  uint16_t getTrknmb() const { return trknmb_; }
 
   /// Set track number (1,2) after sorting ALCTs.
   void setTrknmb(const uint16_t number) { trknmb_ = number; }
 
   /// return 12-bit full BX.
-  int getFullBX() const { return fullbx_; }
+  uint16_t getFullBX() const { return fullbx_; }
 
   /// Set 12-bit full BX.
   void setFullBX(const uint16_t fullbx) { fullbx_ = fullbx; }
 
+  /// return the high multiplicity bits
+  uint16_t getHMT() const { return ((version_ == Version::Run3) ? hmt_ : -1); }
+
+  /// set the high multiplicity bits
+  void setHMT(const int hmt) { version_ == Version::Run3 ? hmt_ = hmt : -1; }
```

Comment:
I think the logic needs to be changed slightly as well, from:
version_ == Version::Run3 ? hmt_ = hmt : -1;
to:
hmt_ = (version_ == Version::Run3 ? hmt : 0);
But it's possible your original if/else logic is valid: I'm not an expert.

Reply:
I would agree, that "hmt_ = (version_..." is better

Reply:
@civanch @abrinke1 Thanks for the suggestion. I changed it to ``` hmt_ = (version_ == Version::Run3) ? hmt : std::numeric_limits<uint16_t>::max();```